bazel/cc_wrapper.py

```diff
@@ -64,6 +64,21 @@ def main():
         argv.append(arg)
   else:
     argv = sys.argv[1:]
+
+  # Add compiler-specific options
+  if "clang" in compiler:
+    # This ensures that STL symbols are included.
+    # See https://github.com/envoyproxy/envoy/issues/1341
+    argv.append("-fno-limit-debug-info")
+
+  if "gcc" in compiler or "g++" in compiler:
```

Comment:
https://github.com/google/oss-fuzz/issues/1337

This is incorrectly adding `-Wno-maybe-uninitialized` when CXX is `clang++`, because `"g++" in "clang++"` is true.